### PR TITLE
Log

### PR DESCRIPTION
**Pull request by Rehan-vulms**
Description
The SignUp component is a reusable front-end component built with React and styled using Tailwind CSS. It provides a minimalistic and user-friendly interface for a sign-up form, allowing users to sign in using Google. It also includes an option for users to subscribe to promotional emails. This component can be integrated into any React-based web application to serve as a login or registration prompt.

Key Features
Responsive Design: The layout is responsive and centered on the page, making it look good on both mobile and desktop devices.
Styled with Tailwind CSS: Tailwind CSS utility classes are used to style the component with minimal custom CSS, ensuring consistent styling with other components.
Google Sign-In Button: A stylized button to encourage sign-in with Google, potentially integrating Google OAuth.
Component Structure
The component consists of the following main sections:

Heading: Displays the title "UploadMint" with "Upload" in white text on a black background and "Mint" in green. This branding title appears prominently at the top of the component.
Form: Contains elements for user interaction:
Subheading: Displays a message, “Sign In to experience ShareMint,” where “ShareMint” is highlighted in green for brand emphasis.
Checkbox: A checkbox labeled “Subscribe to promotional emails” for users to opt in to receive emails.
Sign in with Google Button: A button that encourages users to sign in with Google. It has a hover effect and focus styling for better accessibility.

### CODE REVIEW

File: app/log-in/page.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```
Comment (napstar-420) [inline app/log-in/page.tsx]: no need of this

File: app/log-in/page.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { Button } from '@/components/ui/button';
+
+
+const SignUp: React.FC = () => {
```
Comment (napstar-420) [inline app/log-in/page.tsx]:
```suggestion
export default function Login() {
```

File: app/log-in/page.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  return (
+    <div className="flex items-center justify-center min-h-screen bg-gray-100">
```
Comment (napstar-420) [inline app/log-in/page.tsx]: don't use tailwind colors, use css-variables defined in global.css

File: app/log-in/page.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      <div className="bg-white p-8 rounded-lg shadow-lg w-full max-w-md">
```
Comment (napstar-420) [inline app/log-in/page.tsx]: same here, don't use css variables

File: app/log-in/page.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          >
+            Sign in with Google
+          </Button>
+        </form>
+      </div>
+    </div>
+  );
+};
+
+export default SignUp;
```
Comment (napstar-420) [inline app/log-in/page.tsx]: remove

**Comment by Rehan-vulms** [comment]
Any issues

**Pull request merged**